website/docs/r/trusted_profile_template.html.markdown

```diff
@@ -40,6 +40,103 @@ resource "ibm_iam_trusted_profile_template" "trusted_profile_template_instance"
 }
 ```
 
+Trusted profile template with a Service Id identity and two access policy templates
+
+```hcl
+resource "ibm_iam_trusted_profile_template" "trusted_profile_template_instance" {
+	name = "${var.trusted_profile_template_name}"
+	description = "${var.trusted_profile_template_description}"
+	profile {
+		name = "Profile from Template"
+		description = "description of profile from template"
+		identities {
+			iam_id = "iam-ServiceId-abcd83bd-e218-48af-8073-c0c1b3980001"
+			identifier = "ServiceId-abcd83bd-e218-48af-8073-c0c1b3980001"
+			type = "serviceid"
+		}
+	}
+	policy_template_references {
+		id = split("/", ibm_iam_policy_template.Kube_Administrator_ap_template.id)[0]
+		version = ibm_iam_policy_template.Kube_Administrator_ap_template.version
+  	}
+  	policy_template_references {
+		id = split("/", ibm_iam_policy_template.Services_Reader_ap_template.id)[0]
+		version = ibm_iam_policy_template.Services_Reader_ap_template.version
+  	}
+  	committed = true
+}
+```
+
+Trusted Profile template with service identity and one access policy
+
+```hcl
+resource "ibm_iam_trusted_profile_template" "trusted_profile_template_instance" {
+	name = "${var.trusted_profile_template_name}"
+	description = "${var.trusted_profile_template_description}"
+	profile {
+		name = "Profile from Template"
+		description = "description of profile from template"
+		identities {
+			iam_id = format("crn-%s", var.instance_crn)
```

Comment:
`iam_id` is an optional property which holds redundant information. Should be left out.
Applies to other places later on as well.